dart/lib/src/default_integrations.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:async';
```

Comment:
does it make sense to be `dart_default_integrations.dart ` ? I guess it could conflict to default_integrations.dart  from the flutter package

Reply:
not really, you can''t have filename conflicts, only methods or classes names

Reply:
nice